### ISSUE TITLE
Instructions for customising REST errorHandler

### ISSUE THREAD

**Issue by bajtos** [strongloop/loopback#688]
We need to document instructions for setting a custom `errorHandler` via `server/config.local.js`.

https://github.com/strongloop/strong-remoting/pull/114 implements support for `errorHandler` option, #337 will hopefully modify `loopback.rest` to pass `app.get('remoting').rest` options to the REST handler.

**Comment by crandmck** [comment]
Added http://docs.strongloop.com/display/LB/Environment-specific+configuration#Environment-specificconfiguration-CustomizingRESTerrorhandling.

**Comment by crandmck** [comment]
@bajtos Please review and close if it looks OK.

**Comment by bajtos** [comment]
LGTM.

**Comment by jrschumacher** [comment]
There should be a link to the customizing REST error handling on http://docs.strongloop.com/display/public/LB/Defining+middleware

**Comment by crandmck** [comment]
Added a link to http://docs.strongloop.com/display/LB/Defining+middleware#Definingmiddleware-Error-handlingmiddleware.

**Comment by simoami** [comment]
just for the sake of understanding, we have two error handlers, one referenced in middleware.json and another one in strong-remoting. when does the error handler defined in middleware.json kick in? I tried defining a custom one but it never kicked in. To test, I created a custom API route that explicitly returns an error on callback `cb(err)`. I also used a mix of `throw err` and `1/0` to see if I could trigger the custom error middleware but no luck there. ps I defined the error handler with 4 arguments as express expects.
